javascript-timers/interval.js

```diff
@@ -0,0 +1,18 @@
+const $countDownDisplay = document.querySelector('.countdown-display');
+
+let iD = setInterval(() => {
+  if (iD === 2) {
+    $countDownDisplay.textContent = '3';
+    iD++;
+  } else
+  if (iD === 3) {
+    $countDownDisplay.textContent = '2';
+    iD++;
+  } else
+  if (iD === 4) {
+    $countDownDisplay.textContent = '1';
+    iD++;
+  } else {
+    $countDownDisplay.textContent = '~Earth Beeeelow Us~';
+  }
+}, 1000);
```

Comment:
You shouldn't be listening for specific numbers in your conditionals, you want your code to be dynamic, and work with any number. You should just be setting the `textContent` to whatever the value of the current count is